returnn/util/file_cache.py

```diff
@@ -337,8 +363,12 @@ def _check_existing_copied_file_maybe_cleanup(src_filename: str, dst_filename: s
             return False
         src_stat = os.stat(src_filename)
         dst_stat = os.stat(dst_filename)
-        if src_stat.st_size != dst_stat.st_size:
+        if src_stat.st_size != dst_stat.st_size or src_stat.st_mtime > dst_stat.st_mtime:
             os.remove(dst_filename)
+            try:
+                os.remove(FileCache._get_keepalive_filename(dst_filename))
+            except FileNotFoundError:
+                pass
```

Comment:
Above there's the following code when the keepalive file can't be found:
```
print(
    f"FileCache: Ignoring error file removing keepalive file of {fn}: {type(exc).__name__}: {exc}"
)
```

I'd suggest to uniformize the calls by either setting the print here (recommended), abstracting this into a function (also recommended, as you prefer) or getting rid of the original print and adding a `pass` instead.

Requesting changes for this.